MonoGame.Framework/Utilities/OS.cs

```diff
@@ -0,0 +1,46 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+namespace MonoGame.Utilities
+{
+    /// <summary>
+    /// Defines the operating system.
+    /// </summary>
+    public enum OS
```

Comment:
> What's wrong with the existing platform enumeration used for the content pipeline?

It's per platform instead of per OS. This returns the current OS, and not the platform, WindowsDX, DesktopGL, UWP and Metro can all return Windows as OS.